client/landing/stepper/declarative-flow/flows/onboarding/onboarding.ts

```diff
@@ -66,7 +53,6 @@ const onboarding: Flow = {
 	isSignupFlow: true,
 	__experimentalUseBuiltinAuth: true,
 	useTracksEventProps() {
```

Comment:
This entire hook can be removed as we are no longer tracking/setting these event props in the onboarding flow. This would be best in the site-setup flow where the goal step is now.

Reply:
Continuing the conversation in https://github.com/Automattic/wp-calypso/pull/102424/files#r2030756760